ui/admin/app/components/form/auth-method/ldap/index.hbs

```diff
@@ -76,73 +76,39 @@
-        <Hds::Table class='list-wrapper-field' name='certificates'>
-          <:head as |H|>
-            <H.Tr>
-              <H.Th id={{valueHeaderID}}>{{t 'titles.value'}}</H.Th>
-              <H.Th>{{t 'titles.actions'}}</H.Th>
-            </H.Tr>
-          </:head>
-          <:body as |B|>
-            {{#each @model.certificates as |cert i|}}
-              <B.Tr>
-                <B.Td>
-                  <Hds::Form::Textarea::Field
-                    @value={{cert.value}}
-                    @isOptional={{true}}
-                    disabled={{form.disabled}}
-                    aria-labelledby={{valueHeaderID}}
-                  />
-                </B.Td>
-                <B.Td>
-                  <Hds::Button
-                    @text={{t 'actions.remove'}}
-                    @color='critical'
-                    @icon='trash'
-                    @isIconOnly={{true}}
-                    disabled={{form.disabled}}
-                    {{on 'click' (fn @removeItemByIndex 'certificates' i)}}
-                  />
-                </B.Td>
-              </B.Tr>
-            {{/each}}
-            <B.Tr>
-              <B.Td>
-                <Hds::Form::Textarea::Field
-                  @value={{this.newCertificate}}
-                  @isOptional={{true}}
-                  disabled={{form.disabled}}
-                  aria-labelledby={{valueHeaderID}}
-                />
-              </B.Td>
-              <B.Td>
-                <Hds::Button
-                  @text={{t 'actions.add'}}
-                  @color='secondary'
-                  disabled={{or form.disabled (not this.newCertificate)}}
-                  {{on 'click' this.addCertificate}}
-                />
-              </B.Td>
-            </B.Tr>
-          </:body>
-        </Hds::Table>
-      </F.Control>
-    {{/let}}
-    {{#if @model.errors.certificates}}
-      <F.Error as |E|>
-        {{#each @model.errors.certificates as |error|}}
-          <E.Message>{{error.message}}</E.Message>
-        {{/each}}
-      </F.Error>
-    {{/if}}
-  </Hds::Form::Fieldset>
+  <Form::Field::ListWrapper
+    @layout='horizontal'
+    @isOptional={{true}}
+    @disabled={{form.disabled}}
+  >
+    <:fieldset as |F|>
+      <F.Legend>
+        {{t 'resources.auth-method.form.certificates.label'}}
+      </F.Legend>
+      <F.HelperText>
+        {{t 'resources.auth-method.form.certificates.help'}}
+      </F.HelperText>
+
+      {{#if @model.errors.certificates}}
+        <F.Error as |E|>
+          {{#each @model.errors.certificates as |error|}}
+            <E.Message>{{error.message}}</E.Message>
+          {{/each}}
+        </F.Error>
+      {{/if}}
+    </:fieldset>
+
+    <:field as |F|>
+      <F.Textarea
+        @name='certificates'
+        @options={{@model.certificates}}
+        @addOption={{fn @addOption 'certificates'}}
+        @removeOptionByIndex={{fn @removeOptionByIndex 'certificates'}}
+        @disabled={{form.disabled}}
```

Comment:
I think this `@disabled` arg can be removed since it is already being passed in on line 82

Reply:
thanks for the catch, I updated and cleaned up some of the other files with redundant `@disabled` arg